test/libsolidity/syntaxTests/inlineAssembly/difficulty_reserved.sol

```diff
@@ -0,0 +1,18 @@
+contract C {
+    function f() public view returns (uint ret) {
+        assembly {
+            let difficulty := sload(0)
+            ret := difficulty
+        }
+    }
+    function g() public pure returns (uint ret) {
+        assembly {
+            function difficulty() -> r {
+                r := 1000
+            }
+            ret := difficulty()
+        }
+    }
+}
+// ----
+// ParserError 5568: (98-108): Cannot use builtin function name "difficulty" as identifier name.
```

Comment:
Maybe also `EVMVersion: <paris` here? Is a `difficulty` identifier allowed again for >=paris?

Reply:
If yes, I think the test for this is missing.